insights/parsers/setup_named_chroot.py

```diff
@@ -0,0 +1,54 @@
+"""
+SetupNamedChroot - file ``/usr/libexec/setup-named-chroot.sh``
+==============================================================
+
+This module provides class ``SetupNamedChroot`` for parsing the output of file
+``/usr/libexec/setup-named-chroot.sh``.
+"""
+
+from insights import LegacyItemAccess, Parser, get_active_lines, parser
+from insights.core.filters import add_filter
+from insights.parsers import SkipException
+from insights.specs import Specs
+
+add_filter(Specs.setup_named_chroot, ["ROOTDIR_MOUNT", "/"])
+
+
+@parser(Specs.setup_named_chroot)
+class SetupNamedChroot(Parser, LegacyItemAccess):
```

Comment:
There is another better way better than `LegacyItemAccess` to make the Parser working as a dict:
```python
class SetupNamedChroot(Parser, dict):
  def parser_content():
    ...
    self.update(self.data)
```

Reply:
@xiangce - Acknowledged. Changes done in the latest commit.